packages/zowe-explorer/src/utils/ProfilesUtils.ts

```diff
@@ -45,7 +44,7 @@ export async function errorHandling(errorDetails: any, label?: string, moreInfo?
         "errorHandling.invalid.token",
         "Your connection is no longer active. Please log in to an authentication service to restore the connection."
     );
-    [errorDetails, errorDetails.mDetails, label, moreInfo].filter(Boolean).forEach(globals.LOG.error);
+    globals.LOG.error(`${errorDetails}\n` + JSON.stringify({ errorDetails, label, moreInfo }));
```

Comment:
Do we need `errorDetails` twice here? Or should this be `${errorDetails.message}`? If neither then you can disregard this comment

Reply:
My goal was to log as much info about the error as possible. I believe `${errorDetails}` is identical to `${errorDetails.message}` because the `toString` property for errors returns `error.message`. The reason I also included the error in the `JSON.stringify` is that some errors like instances of ImperativeError contain additional properties that may be useful to see in logs.

Reply:
Ah! I figured that was an oversight on my behalf - didn't realize that their `toString` method returned the message parameter. No worries, everything LGTM 👍